recipes-bsp/txe-secure-boot/txesbmantool_git.bb

```diff
@@ -0,0 +1,42 @@
+SUMMARY = "Utility for generating TXE Secure Boot manifests"
```

Comment:
Looks like it comes from coreboot/utils repo, so can be also moved to meta-coreboot/recipes-bsp/coreboot-utils

Reply:
removed from meta-dasharo: https://github.com/Dasharo/meta-dasharo/pull/4/commits/140c8ab31add90e9ce5712f658862d11909d3563
added to meta-coreboot: https://github.com/zarhus/meta-coreboot/pull/2/commits/3e86c02a21812e52d5d7acbef96e002154dc92c1